File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.js
function solution(sides) {
  let result = 0;

  const max = Math.max(...sides);
  const min = Math.min(...sides);

  for (let i = max - min + 1; i <= max; i++) {
    result += 1;
  }

  for (let j = max + 1; j < min + max; j++) {
    result += 1;
  }
  
  return result
}

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.java
class Solution {
    public int solution(int[] sides) {
        int answer = 0;
        int max = Math.max(sides[0], sides[1]); // 가장 긴 변
        int min = Math.min(sides[0], sides[1]); // 가장 짧은 변

        answer += min * 2 - 1;

        return answer;
    }
}